templates/systemd-override.erb

```diff
@@ -1,4 +1,8 @@
+<% if @managed_package_repo -%>
+  .include /lib/systemd/system/postgresql-<%= @version %>.service
```

Comment:
is this the same across all versions (8.x - 9.4, or whatever we support), or is it a new addition?